server/app/views/ViewUtils.java

```diff
@@ -396,6 +398,36 @@ public static DivTag makeAlert(
                 .with(p().withClass("usa-alert__text").withText(text)));
   }
 
+  /**
+   * Makes a USWDS Alert component with the given text and optional title. Alert variant is
+   * determined by the {@link AlertType} passed in.
+   * https://designsystem.digital.gov/components/alert/
+   *
+   * @param text The text to include in the alert.
+   * @param hidden Whether or not to set the hidden property on the component.
+   * @param title An optional title to be included in the alert.
+   * @param alertType The type of {@link AlertType} alert to show
+   * @param classes One or more additional classes to apply to the USWDS Alert component.
+   * @return DivTag containing the alert.
+   */
+  public static DivTag makeAlert(
```

Comment:
Question/suggestion: is an alert type required for all alerts? If so I think it would be preferable to make the makeAlert method above private and refactor the existing usages to use this new one. It's confusing to have two ways to accomplish the same thing (i.e. pass the alert type in the class list or call this method).

Relatedly, is the makeAlertSlim method kind of redundant then too? Actually I see calls to makeAlertSlim that also pass BaseStyles.ALERT_INFO so maybe an alert can have more than one style?! Or is that a bug? 🤔 

If all the above is possible, then would it make sense to move the ALERT_ strings out of BaseStyles so that one is not tempted to use them extra-curricularly (outside of this method)?

Reply:
I thought about doing that, but I wasn't sure if there were future cases Melanie had in mind when she created it. I was going to revisit it down the road after she gets back from vacation.